uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/RequestParameterPolicyEnforcementFilter.java

```diff
@@ -30,7 +30,7 @@
  * do not meet configured rules.
  *
  * <p>This is a fork of the Filter offered at
- * https://github.com/Jasig/cas-server-security-filter/pull/6 .
+ * https://github.com/uPortal-Project/cas-server-security-filter/pull/6 .
```

Comment:
This repo is still in the Jasig Github org.

Reply:
Thanks Jonathan, I fixed it with apereo link (removing the redirect from Jasig)